crm_won_reason/models/crm_lead.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class CrmLead(models.Model):
+    _inherit = "crm.lead"
+
+    won_reason_id = fields.Many2one("crm.lost.reason")
```

Comment:
i dont remember much after ~1 yr but looking at the card issue was that there was only possibility to define lost reasons, so idea was to have selection field Reason Type (lost or won) so we could assign proper value to any record on lost.reason model (i agree it is stupid because model name is crm.lost.reason...). With this we treated this model as both won&lost reason. Then on lead we would have 2 buttons (won/lost) and wizard where we can choose reason (filtered as lost, won, depending on button used).

cc @yankinmax

Reply:
there is still no separate `won.reason` model, and it's not needed